browser-sign-in/OktaBrowserSignIn/WelcomeViewController.swift

```diff
@@ -15,70 +15,88 @@
  */
 
 import UIKit
-import OktaOidc
+import WebAuthenticationUI
 
 final class WelcomeViewController: UIViewController {
-
-    var oktaOidc: OktaOidc?
-    var stateManager: OktaOidcStateManager?
+    var auth: WebAuthentication?
```

Comment:
Similar to the other comments, it might be better to directly access `WebAuthentication.shared` and guard.